File: ts/test/exists.test.ts

import { test, describe } from 'node:test'
import { equal } from 'node:assert'


import { Cie10SDK } from '..'


describe('exists', async () => {

  test('test-mode', async () => {
    const testsdk = await Cie10SDK.test()
    equal(null !== testsdk, true)
  })

})

File: lua/test/exists_test.lua
-- Cie10 SDK exists test

local sdk = require("cie10_sdk")

describe("Cie10SDK", function()
  it("should create test SDK", function()
    local testsdk = sdk.test(nil, nil)
    assert.is_not_nil(testsdk)
  end)
end)
